source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -317,6 +327,36 @@ namespace
         return totalDurations;
     }
 
+    void benchmarkAssistedCapture3D(Zivid::Camera &camera, const size_t numFrames)
+    {
+        printAssistedCapture3DHeader(numFrames);
+
+        Zivid::CaptureAssistant::SuggestSettingsParameters suggestSettingsParameters(
+            std::chrono::milliseconds{ 1200 }, Zivid::CaptureAssistant::AmbientLightFrequency::none);
+
+        for(size_t i = 0; i < 5; i++) // Warmup
+        {
+            const auto settingsVector{ Zivid::CaptureAssistant::suggestSettings(camera, suggestSettingsParameters) };
+        }
```

Comment:
is it required to run warmup on this specifically? If so, why is it sufficient?

Reply:
The first capture is always much slower here, see the image below. The second capture can be slightly slower (e.g. 880 for 820 average), but this one I saw only once in 15-ish tests.

The observation is similar for 3D captures, but for them, occasionally, a number of first 5 frames can be outliers also (this happens more commonly than above).

I wanted to keep this warmup consistent with the ones in other functions where we use x5, benchmarkCapture2D() is the most similar case to this one and it also uses 5.

Alternatively, we could use 1 or 2 warmup captures for 2D captures and for this. What do you think?

![image](https://user-images.githubusercontent.com/46567886/79831690-2fec9680-83a8-11ea-85d2-12829f159c86.png)